
collision_radius: 0.4 # Radius for collision
collision_range: 10 # Range for collision dectection
collision_acceleration: 5 # Acceleration
time_collision_penality: 100.0 # Time penalty
pos_uncertain_penalty: 1
vel_uncertain_penalty: 1
num_points_admissible: 10

range: 2.0

solve_order: 5

gps_position_stdev: 0.0
gps_velocity_stdev: 0.0

drop_probability: 0 # Values between 0 and 100

sigmaQ_vel: 3
alphaQ_vel: 0.5
sigmaQ_jrk: 0.2
alphaQ_jrk: 0.5

sigmaR_pos: 0.0
sigmaR_vel: 0.0

add_uncertainty: false

add_kalman: false

radar_measurement: false

sigmaR_range: 0.6
sigmaR_zenith: 0.01745
radar_x: 0
radar_y: -200


buffer_on: true
buffer_power: 50

dt: 0.1
